mep/common/management/export.py

```diff
@@ -94,6 +94,8 @@ def handle(self, *args, **kwargs):
         # get stream array / generator of data for export
         data = self.get_data(kwargs.get("max"))
         self.stdout.write("Exporting JSON and CSV")
+        # ensure directory exists (useful to allow command line user to specify dated dir)
+        os.makedirs(os.path.dirname(base_filename), exist_ok=True)
```

Comment:
I always assume directory exists or I make it myself, but this would be more convenient. Worth briefly noting in the help text?